scripts/tests/AUTH/test_3d_phase3_config_integrity.py

```
@@ -0,0 +1,487 @@
+        FROM deployment_config
+        WHERE key = '{escaped_key}'
+        LIMIT 1;
+        """,
+        db_path,
+    )
+    return output if output else None
+
+
+def test_secret_encrypted_at_rest(api_base: str, headers: dict[str, str], db_path: str) -> bool:
+    print("\n[TEST 3D.1] Secret-at-rest encryption")
+    secret_key = "SMTP_PASS"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "secret_key"

(S105)